src/System.Resources.Reader/src/System.Resources.Reader.csproj

```diff
@@ -17,6 +18,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System\Resources\ResourceReader.cs" />
+    <Compile Include="System\Resources\StreamExtensions.cs" />
```

Comment:
Yeah, I think we need a set of APIs like that, but I would do it as a separate project. I will file a bug to track it.